mongo/index_view_internal_test.go

```diff
@@ -87,6 +87,25 @@ func getIndexableCollection(t *testing.T) (string, *Collection) {
 	return dbName, db.Collection(dbName)
 }
 
+// Helper utility added in scope of continuous matrix testing work to wrap
+// getServerVersion() and compareVersions().
+func getAndCompareVersion(t *testing.T, targetVersion string) int {
+	t.Helper()
+
+	serverVersion, err := getServerVersion(createTestDatabase(t, nil))
+	require.NoError(t, err)
+	return compareVersions(t, serverVersion, targetVersion)
+}
+
+func assertIndexNSMatch(t *testing.T, expected, actual string) {
+	t.Helper()
+
+	// 4.4+ servers do not return an "ns" field in index specifications.
```

Comment:
Perhaps I'm misunderstanding CMT, but do these tests get run against a 4.4+ server? I thought this branch would only be tested against a 4.2 mdb instance.

Reply:
The 4.2 driver was already tested against the 4.2 server when we were developing it. The goal of this project is to test "old" drivers against "new" servers to ensure that customer applications running with an older driver version like this one will continue to function if the server is upgraded. To do this, we test driver version X against all server versions > X. For example, 4.0 drivers get tested against 4.2, 4.4, and latest.

Reply:
Ah that makes sense; the 4.2 branch just represents the state of the driver as of server version 4.2. Thanks for the explanation.